src/util/pelz_io.c

```diff
@@ -41,29 +41,35 @@ void ocall_free(void *ptr, size_t len)
   free(ptr);
 }
 
-ExtensionType get_file_ext(charbuf buf)
+ExtensionType get_file_ext(char *filename)
```

Comment:
Should this function separate ```NULL``` filename from unrecognized extension from no extension? As written it appears all 3 return ```NO_EXT```, which is then assumed to contain a plaintext key in ```key_load```.

Reply:
The function is checking if the ```filename``` has a ```.nkl``` or a ```.ski``` extension. The ```NO_EXT``` value is just a place holder to let you know it is does not have ```.nkl``` or ```.ski```.  It is not informing you if the ```filename``` is valid or if it has another extension type but just if it has one of the two values being checked.  So I do not think it need to be changed unless we want to add that clarity to ```ExtensionType```.